doc/source/examples/cf_mnist.ipynb

```diff
@@ -22,6 +22,20 @@
     "In this notebook we illustrate the usage of the basic counterfactual algorithm on the MNIST dataset."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "To enable support for Counterfactual, you may need to run\n",
+    "    \n",
+    "```bash\n",
+    "pip install alibi[tensorflow]\n",
+    "```\n",
+    "\n",
+    "</div>"
```

Comment:
Same comment as above, can we make the box title be "Note" and the text part of the body?